.github/actions/generate-chart-matrix-nightly/action.yml

```diff
@@ -0,0 +1,71 @@
+name: Generate chart and Enabled Scenarios Matrix
+description: Find all scenarios for all charts and list charts with enabled scenarios.
+inputs:
+  charts-path:
+    description: Charts path to match.
+    default: "charts/camunda-platform*"
+  type:
+    description: what type of integration tests do you want to run? e.g pr, nightly
+    default: pr
+outputs:
+  matrix:
+    description: JSON matrix of changed charts which will be used as and input for GHA workflow matrix.
+    value: ${{ steps.set-chart-versions.outputs.matrix }}
+runs:
+  using: composite
+  steps:
+    - name: ℹ️ Print workflow inputs ℹ️
+      shell: bash
+      env:
+        GITHUB_CONTEXT: ${{ toJson(inputs) }}
+      run: |
+        echo "Action Inputs:"
+        echo "${GITHUB_CONTEXT}"
+    - name: Generate matrix
+      shell: bash
+      id: set-chart-versions
+      run: |
+        touch matrix_versions.txt
+        echo "matrix:" >>matrix_versions.txt
+        ls -d charts/camunda-platform-* | while read chart_dir; do
+          camunda_version="$(basename ${chart_dir} | sed 's/camunda-platform-//')"
+
+          # skip chart version with "alpha" in the chrt version
+          case "$camunda_version" in
```

Comment:
The case here is unnecessary, just an if condition.

Reply:
I added this if statement in because this command would pick up all chart directories. Even symlink directories:
```
ls -d charts/camunda-platform*
```

You can see for yourself when running this command locally

Reply:
removed the case and used if statement